aggregator/aggregator.go

```diff
@@ -289,33 +219,13 @@ func (a *Aggregator) handleReorg(reorgData synchronizer.ReorgExecutionResult) {
 func (a *Aggregator) handleRollbackBatches(rollbackData synchronizer.RollbackBatchesData) {
 	a.logger.Warnf("Rollback batches event, rollbackBatchesData: %+v", rollbackData)
 
-	a.streamClientMutex.Lock()
-	defer a.streamClientMutex.Unlock()
-
-	dsClientWasRunning := a.streamClient.IsStarted()
-
 	var err error
 
-	if dsClientWasRunning {
-		// Disable the process entry function to avoid processing the data stream
-		a.streamClient.ResetProcessEntryFunc()
-
-		// Stop Reading the data stream
-		err = a.streamClient.ExecCommandStop()
-		if err != nil {
-			a.logger.Errorf("failed to stop data stream: %v.", err)
-		} else {
-			a.logger.Info("Data stream client stopped")
-		}
-	}
-
 	// Get new last verified batch number from L1
 	var lastVerifiedBatchNumber uint64
-	if err == nil {
-		lastVerifiedBatchNumber, err = a.etherman.GetLatestVerifiedBatchNum()
-		if err != nil {
-			a.logger.Errorf("Error getting latest verified batch number: %v", err)
-		}
+	lastVerifiedBatchNumber, err = a.etherman.GetLatestVerifiedBatchNum()
+	if err != nil {
```

Comment:
Nitpick, simplification:

```suggestion

	// Get new last verified batch number from L1
	lastVerifiedBatchNumber, err := a.etherman.GetLatestVerifiedBatchNum()
```